backend/src/node/connector.rs

```diff
@@ -22,14 +28,17 @@ pub struct NodeConnector {
     hb: Instant,
     /// Aggregator actor address
     aggregator: Addr<Aggregator>,
-    /// Chain actor address
-    chain: Option<Addr<Chain>>,
+    /// Mapping message connection id to addresses of chains for multiplexing
+    /// a node running multiple parachains
+    chains: BTreeMap<ConnId, Addr<Chain>>,
     /// Backlog of messages to be sent once we get a recipient handle to the chain
-    backlog: Vec<NodeMessage>,
+    backlogs: BTreeMap<ConnId, Vec<NodeMessage>>,
```

Comment:
Unrelated to this PR, but is it ok that the backlog is unbounded? Maybe we could re-work this into a bounded queue?
EDIT: I see it is bounded to `10` below. Let's document that.

Reply:
Yeah, it's bounded. I'm more worried about the number of `ConnId` being unbounded atm.